subcommand/acl-controller/command_ent_test.go

```diff
@@ -42,15 +42,12 @@ func TestUpsertConsulClientTokenEnt(t *testing.T) {
 	testUpsertConsulClientToken(t, cases)
 }
 
-func TestCreatePartitionEnt(t *testing.T) {
+func TestUpsertPartitionEnt(t *testing.T) {
 	cases := map[string]struct {
 		partition       string
 		createPartition bool
 		err             error
 	}{
-		"when partitions are not enabled": {
-			partition: "",
-		},
```

Comment:
These tests weren't running in CI because the enterprise build flag wasn't being passed! I fixed that in the Circle CI config. Also, this case is removed, since the ["skip partition creation" logic](https://github.com/hashicorp/consul-ecs/blob/d4f5e2b1bbc565e990bb1c1cd3a55f4822e2eaea/subcommand/acl-controller/command.go#L97-L107) is outside the `upsertPartition` function.